engine/sync_test.go

```diff
@@ -136,20 +125,54 @@ func TestSyncHeartbeats(t *testing.T) {
 	go server.run()
 	go client.run()
 
+	// TODO: Test we can disable and re-enable the client
```

Comment:
Seems this TODO is already well covered (re-enabled in line 144 after disabled in line 140). If there's something else this TODO is talking about, could you expand the description a little bit more?

Reply:
I'm totally unsure. Removed.